pkg/registry/core/endpoint/strategy.go

```diff
@@ -60,8 +59,6 @@ func (endpointsStrategy) Validate(ctx context.Context, obj runtime.Object) field
 
 // Canonicalize normalizes the object after validation.
 func (endpointsStrategy) Canonicalize(obj runtime.Object) {
-	endpoints := obj.(*api.Endpoints)
-	endpoints.Subsets = endptspkg.RepackSubsets(endpoints.Subsets)
```

Comment:
I'm fine with that I described it already before, but it would be good if @thockin could confirm it.

Reply:
We talked about this today as part of sig-network and the general consensus from @thockin and others seemed to be that we should remove this.

Reply:
Thanks - resolving then.